modules/_node-scripts/bin.js

```diff
@@ -84,7 +84,8 @@ const COMMANDS = {
 
 		See this help's introduction to find the meaning of --all, --current-branch, ... parameters.
 `,
-		parameters: '[--check] [{--all|--current-branch|--local-changes}]',
+		parameters:
+			'[--check] [{--all|--current-branch|--local-changes|--emit-all}]',
```

Comment:
Will people understand what `--emit-all` means (vs, for example, `--all`) ? :thinking: 

Above, in line 72, the help message says:

> See this help's introduction to find the meaning of --all, --current-branch, ... parameters.

Maybe we should add the meaning of `--emit-all` to that description...

Reply:
Also, is `--emit-all` intended to be used with one of the others? :thinking: 

Because if that's the case, then the help message should be something like:

`[--check] [--emit-all] [{--all|--current-branch|--local-changes}]

Or maybe `--emit-all` implies `--all`? :thinking: